.circleci/config.yml

```diff
@@ -0,0 +1,66 @@
+version: 2.1
+executors:
+  infrastructure_container:
+    docker:
+      - image: aeternity/infrastructure
```

Comment:
:latest tag so master branch?

Reply:
Or :stable?

Reply:
I prefer to integrate with latest changes, as all of the modules have a lot of dynamics.